utils/bash/patches/100-fix-jobs.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/jobs.c b/jobs.c
```

Comment:
Please use git-format-patch style patch with commit messages included.  Thank you

Reply:
Is this the hold up? Unfortunately I have no idea how to do what you're asking for.

Reply:
Something like the original source is doing: https://bugs.alpinelinux.org/issues/8447.  Preferably with additional note that the patch is a port from that source

Reply:
Bash in my first impression is an ancient and complex piece of software.  It's subtle and tricky.  If we do not take time to do proper annotation and documentation now, we are making it hard for future reviewers and potential maintainers, essentially wasting their time.

Reply:
It's mentioned/referenced in commit message and I have no idea what you want me to put there or how.